plugin/Controller/AdminViewController.php

```diff
@@ -148,7 +148,7 @@ public function register_menu()
 		add_menu_page( __('onOffice', 'onoffice-for-wp-websites'), __('onOffice', 'onoffice-for-wp-websites'),
 			$roleMainPage, $this->_pageSlug, function() use ($pDI) {
 				echo $pDI->get(MainPage::class)->render();
-			}, 'dashicons-admin-home');
+			}, plugin_dir_url(ONOFFICE_PLUGIN_DIR.'/index.php').'images/onOffice-Logo-screen_wp-grey.svg');
```

Comment:
If this works, the behavior is not documented: https://developer.wordpress.org/reference/functions/add_menu_page/#parameters

It would be safer to use the base64 variant.